### PR TITLE
Update Startup.cs

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Contributes to [this internal issue](https://github.com/dotnet/aspnetcore-internal/issues/4522#issuecomment-2078104865) and is required to be obvious in query results.

image

### CODE REVIEW

**Comment by Rick-Anderson** [comment]
@tdykstra  can you get this merged?

**Review by tdykstra — APPROVED** [review]

**Comment by tdykstra** [comment]
> @tdykstra can you get this merged?

I don't have the required permissions.

File: samples/csharp/end-to-end-apps/Forecasting-Sales/src/eShopDashboard/Startup.cs [hunk 1]
```diff
@@ -97,7 +97,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             app.UseSwagger();
             if (app.Environment.IsDevelopment())
```
Comment (captainsafia) [inline samples/csharp/end-to-end-apps/Forecasting-Sales/src/eShopDashboard/Startup.cs:98]:
```suggestion
            if (env.IsDevelopment())
```

**Review by luisquintanilla — APPROVED** [review]
LGTM

**Comment by Rick-Anderson** [comment]
@luisquintanilla  can you merge the PR?

**Pull request merged**